exercises/sum.cpp

```diff
@@ -6,3 +6,24 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream>
+using namespace std;
+
+int main() {
+    int num1, num2, sum;
+
+    // prendo in input i due numeri
+    cout << "Insert the first number: ";
+    cin >> num1;
+    cout << "Insert the second number: ";
+    cin >> num2;
+
+    // sommo i due numeri
+    sum = num1 + num2;
```

Comment:
andrebbe messo in una funzione l'operazione somma al fine di riutilizzarla, magari mediante template